docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Concatenate.cs

```diff
@@ -9,38 +9,56 @@ public static class Concatenate
             {
-                new InputData(){ Feature1 = 0.1f, Feature2 = new[]{ 1.1f, 2.1f, 3.1f}, Feature3 = 1 },
-                new InputData(){ Feature1 = 0.2f, Feature2 = new[]{ 1.2f, 2.2f, 3.2f}, Feature3 = 2 },
-                new InputData(){ Feature1 = 0.3f, Feature2 = new[]{ 1.3f, 2.3f, 3.3f}, Feature3 = 3 },
-                new InputData(){ Feature1 = 0.4f, Feature2 = new[]{ 1.4f, 2.4f, 3.4f}, Feature3 = 4 },
-                new InputData(){ Feature1 = 0.5f, Feature2 = new[]{ 1.5f, 2.5f, 3.5f}, Feature3 = 5 },
-                new InputData(){ Feature1 = 0.6f, Feature2 = new[]{ 1.6f, 2.6f, 3.6f}, Feature3 = 6 },
+                new InputData(){ Feature1 = 0.1f, Feature2 = new[]{ 1.1f, 2.1f,
+                    3.1f }, Feature3 = 1 },
+
+                new InputData(){ Feature1 = 0.2f, Feature2 = new[]{ 1.2f, 2.2f,
+                    3.2f }, Feature3 = 2 },
+
+                new InputData(){ Feature1 = 0.3f, Feature2 = new[]{ 1.3f, 2.3f,
+                    3.3f }, Feature3 = 3 },
+
+                new InputData(){ Feature1 = 0.4f, Feature2 = new[]{ 1.4f, 2.4f,
+                    3.4f }, Feature3 = 4 },
+
+                new InputData(){ Feature1 = 0.5f, Feature2 = new[]{ 1.5f, 2.5f,
+                    3.5f }, Feature3 = 5 },
+
+                new InputData(){ Feature1 = 0.6f, Feature2 = new[]{ 1.6f, 2.6f,
+                    3.6f }, Feature3 = 6 },
             };
 
             // Convert training data to IDataView.
             var dataview = mlContext.Data.LoadFromEnumerable(samples);
 
-            // A pipeline for concatenating the "Feature1", "Feature2" and "Feature3" columns together into a vector that will be the Features column.
-            // Concatenation is necessary because trainers take feature vectors as inputs.
+            // A pipeline for concatenating the "Feature1", "Feature2" and
+            // "Feature3" columns together into a vector that will be the Features
+            // column. Concatenation is necessary because trainers take feature
+            // vectors as inputs.
             //
-            // Please note that the "Feature3" column is converted from int32 to float using the ConvertType.
-            // The Concatenate requires all columns to be of same type.
-            var pipeline = mlContext.Transforms.Conversion.ConvertType("Feature3", outputKind: DataKind.Single)
-                .Append(mlContext.Transforms.Concatenate("Features", new[] { "Feature1", "Feature2", "Feature3" }));
+            // Please note that the "Feature3" column is converted from int32 to
+            // float using the ConvertType. The Concatenate requires all columns to
+            // be of same type.
+            var pipeline = mlContext.Transforms.Conversion.ConvertType("Feature3",
+                outputKind: DataKind.Single)
+                .Append(mlContext.Transforms.Concatenate("Features", new[]
+                    { "Feature1", "Feature2", "Feature3" }));
 
```

Comment:
Extra spaces on line 52

Reply:
intentional to highlight parameter of the append method